### PR TITLE
Add 24 descriptions (2 of 6)

### PR DESCRIPTION
**Pull request by rasa**

### CODE REVIEW

File: bucket/git-with-openssh.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "##": "Maintainers: when updating this manifest to a new version, you might like to also update git.json",
     "homepage": "https://git-for-windows.github.io/",
+    "description": "A free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.",
```
Comment (niheaven) [inline bucket/git-with-openssh.json]:
```suggestion
    "description": "A free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. (w/ built-in OpenSSH)",
```

File: bucket/gof.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://github.com/mattn/gof",
+    "description": "Fuzzy search tool written in Go",
```
Comment (Ash258) [inline bucket/gof.json]:
```suggestion
    "description": "Fuzzy search tool written in Go.",
```

File: bucket/handbrake-cli.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://handbrake.fr/",
+    "description": "A tool for converting video from nearly any format to a selection of modern, widely supported codecs.",
```
Comment (Ash258) [inline bucket/handbrake-cli.json]:
```suggestion
    "description": "An video transcoder with multiple formats support.",
```

File: bucket/gradle.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://gradle.org",
+    "description": "An open-source build automation tool focused on flexibility and performance. ('all' version)",
```
Comment (Ash258) [inline bucket/gradle.json]:
```suggestion
    "description": "An open-source build automation tool focused on flexibility and performance. (Source code and documentation boundled)",
```

File: bucket/gradle-bin.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://gradle.org",
+    "description": "An open-source build automation tool focused on flexibility and performance. ('bin' version)",
```
Comment (Ash258) [inline bucket/gradle-bin.json]:
```suggestion
    "description": "An open-source build automation tool focused on flexibility and performance. (Binaries only without documentation)",

```

**Comment by niheaven** [comment]
> Note that the QuantumDepth=8 version (Q8) which provides industry standard 24/32 bit pixels consumes half the memory and about 30% less CPU than the QuantumDepth=16 version (Q16) which provides 48/64 bit pixels for high-resolution color. A Q8 version is fine for processing typical photos intended for viewing on a computer screen. If you are dealing with film, scientific, or medical images, use ICC color profiles, or deal with images that have limited contrast, then the Q16 version is recommended.

I think keep q8 here (and drop `-q8`) and move q16 to `versions` should be better.

File: bucket/gradle-bin.json [hunk 6]
```diff
@@ -1,5 +1,7 @@
 {
     "homepage": "https://gradle.org",
+    "description": "An open-source build automation tool focused on flexibility and performance. (Binaries only without documentation)",
+
```
Comment (Ash258) [inline bucket/gradle-bin.json]: Sorry

```suggestion
```

**Pull request merged**